azure-toolkit-libs/azure-toolkit-auth-lib/src/main/java/com/microsoft/azure/tools/auth/exception/AzureLoginException.java

```diff
@@ -0,0 +1,13 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
+
+package com.microsoft.azure.tools.auth.exception;
+
+public class AzureLoginException extends Exception {
```

Comment:
Do we need to have a parent class like `AzureException`

Reply:
this is auth lib, all auth exceptions should inherit this exception